### PR TITLE
[DO NOT MERGE] Differences between main and random_fid

### PR DESCRIPTION
**Pull request by alexhernandezgarcia**

### CODE REVIEW

File: config/random_sampler.yaml
Comment (alexhernandezgarcia) [inline config/random_sampler.yaml:1]: Should this file be in main?

**Comment by alexhernandezgarcia** [comment]
@nikita-0209 The goal of this PR is to try to figure out what things of branch `rand_fid` are specific / important to run experiments with random fidelities. If you know what these things are, please flag them.

File: storage/model_oracle/oracle_MLP
Comment (alexhernandezgarcia) [inline storage/model_oracle/oracle_MLP:1]: What was this?

File: sampler/random_grid.py
Comment (alexhernandezgarcia) [inline sampler/random_grid.py:1]: Should this be in main?

File: storage/model_oracle/oracle_MLP
Comment (nikita-0209) [reply storage/model_oracle/oracle_MLP:1]: supposed to be the AMP oracle

**Comment by nikita-0209** [comment]
> @nikita-0209 The goal of this PR is to try to figure out what things of branch `rand_fid` are specific / important to run experiments with random fidelities. If you know what these things are, please flag them.

Okay I am not sure I understand what is going on here. Why are we merging main into `rand_fid`?
It would probably be helpful to push my local changes of rand_fid to the current rand_fid first? All changes in the local `rand_fid` are crucial to run the random fidelity experiments.

**Pull request closed without merging**